src/Multiplier.php

```diff
@@ -462,6 +462,7 @@ public function resetFormEvents(): void
 	/**
 	 * @param string|null $returnType
 	 * @return object|mixed[]
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
```

Comment:
Why not to fix this issue?

Reply:
Nette\Forms\Container - getValues() - typehint is not specified there https://github.com/nette/forms/blob/master/src/Forms/Container.php#L108